tests/entrypoints/openai/test_token_in_token_out.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+
+import pytest
+
+from vllm.model_executor.model_loader.weight_utils import (
+    download_weights_from_hf)
+from vllm.transformers_utils.tokenizer import get_tokenizer
+
+from ...utils import RemoteOpenAIServer
+
+MODEL_NAME = "Qwen/Qwen3-0.6B"
+MODEL_PATH = "/tmp/qwen3_06b"
+
+
+@pytest.fixture(scope="module")
+def server():
+    global MODEL_PATH
+    MODEL_PATH = download_weights_from_hf(
+        MODEL_NAME,
+        allow_patterns=["*"],
+        cache_dir="/tmp/qwen3_06b",
+        ignore_patterns=["tokenizer*", "vocab*"])
```

Comment:
also ignore the safetensors to accelerate test time.

Reply:
ignoring safetensors will cause failure, unless we set load format to be dummy.
> RuntimeError: Cannot find any model weights with `/tmp/qwen3_06b/models--Qwen--Qwen3-0.6B/snapshots/c1899de289a04d12100db370d81485cdf75e47ca`